src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -314,7 +314,9 @@ def load_arguments(self, _):
                    "managed identities. For managed identities use the principal id. For service principals, "
                    "use the object id and not the app id.")
         c.argument('ids', nargs='+', help='space-separated role assignment ids')
-        c.argument('include_classic_administrators', arg_type=get_three_state_flag(), help='list default role assignments for subscription classic administrators, aka co-admins')
+        c.argument('include_classic_administrators', arg_type=get_three_state_flag(),
+                   help='list default role assignments for subscription classic administrators, aka co-admins',
+                   deprecate_info=c.deprecate(target='--include-classic-administrators'))
```

Comment:
Without `target='--include-classic-administrators'`, the warning message will be:

> Argument 'include_classic_administrators' has been deprecated and will be removed in a future release.